lib/apia/open_api/objects/path.rb

```diff
@@ -38,12 +38,14 @@ def initialize(spec:, path_ids:, route:, name:, api_authenticator:)
             operationId: convert_route_to_id,
             summary: @route.endpoint.definition.name,
             description: @route.endpoint.definition.description,
-            tags: route.group ? get_group_tags(route.group) : [name]
+            tags: route.group ? get_group_tags(route.group) : [name],
+            security: []
```

Comment:
i guess this is creating an empty array for endpoints that don't use scopes. is that what we want? Just wonder if it bloats the spec quite a bit. Perhaps it's better to be explicit that none are required with an empty array, rather than omitting it entirely though.